src/routes/namespaces/[namespace]/workflows/_workflows-summary-table.svelte

```diff
@@ -1,18 +1,17 @@
-<table class="border-collapse w-full">
-  <thead>
-    <tr class="bg-gray-200">
-      <th class="w-3/12 text-left ">Workflow Name</th>
-      <th class="w-3/12 text-left">Workflow ID</th>
-      <th class="w-1/12">Status</th>
-      <th class="w-2/12 text-right">Started</th>
-      <th class="w-2/12 text-right">Ended</th>
-    </tr>
-  </thead>
-  <slot name="rows" />
-</table>
+<section class="workflow-table border-collapse w-full rounded-lg">
```

Comment:
The table itself should probably have a border along the sides. Right now, it's missing a border along the bottom entirely. I suggest adding the following classes: `border-2 border-gray-300 overflow-hidden`